app/views/friend_requests/sent_request.html.erb

```diff
@@ -5,13 +5,13 @@
   
     <div class="col-8">
       <div class="card">
-        <% if current_user.pending_requests_to_users.count.positive? %>
+        <% if current_user.outgoing_pending_friends.count.positive? %>
```

Comment:
**[REQUIRED]**
* Let's separate code logic from the views([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright).